crates/test-collector/src/lib.rs

```diff
@@ -277,6 +277,7 @@ pub struct LinkedLibrary {
 pub struct TestCase {
     pub name: String,
     pub available_gas: Option<usize>,
+    pub expectation: TestExpectation,
```

Comment:
```suggestion
    pub expected_result: TestExpectation,
```
I think it makes a better name, purpose is more obvious. Could you also rename `TestExpectation` -> `ExpectedTestResult` `PanicExpectation` -> `ExpectedPanicValue` ? I know those existed before, but i believe renaming those would be an improvement.

Reply:
done